explainer.md

```diff
@@ -0,0 +1,76 @@
+# Screen Capture Explained
+
+## What is it? What’s it for?
+
+It’s an API to get a live video stream of your computer screen or a single window on it, optionally with sound.
+
+It’s sometimes referred to it as screen-sharing, even though it’s only the capturing part being defined (sharing is just the primary use case).
+
+Have you ever been a presenter in an online video conference call? You probably shared your screen or the window of a presentation-program to do that. Have you ever presented in an online video conference call without this feature? That would probably require printouts and hand-waving! It’s a pretty key feature. This is that feature.
```

Comment:
If content is online, it is possible to have everyone look at their own copy of that content, but that can lead to confusion as everyone can get a subtly different view.